src/main/java/spireMapOverhaul/zones/brokenSpace/relics/BrokenDataDisk.java

```diff
@@ -0,0 +1,66 @@
+package spireMapOverhaul.zones.brokenSpace.relics;
+
+import com.evacipated.cardcrawl.mod.stslib.relics.OnChannelRelic;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.RelicAboveCreatureAction;
+import com.megacrit.cardcrawl.actions.defect.ChannelAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.orbs.AbstractOrb;
+import com.megacrit.cardcrawl.powers.FocusPower;
+import com.megacrit.cardcrawl.relics.DataDisk;
+
+import static spireMapOverhaul.util.Wiz.adp;
+
+public class BrokenDataDisk extends BrokenRelic implements OnChannelRelic {
+    public static final String ID = "BrokenDataDisk";
+    public static final int AMOUNT = 1;
+    public boolean shouldTrigger = true;
+    private boolean usedThisTurn = false;
+
+    public BrokenDataDisk() {
+        super(ID, RelicTier.SPECIAL, LandingSound.CLINK, DataDisk.ID);
+    }
+
+    @Override
+    public void onChannel(AbstractOrb abstractOrb) {
+        if (!shouldTrigger || usedThisTurn) {
+            return;
+        }
+
```

Comment:
You should add a `this.flash();` since that's the standard way to show a relic activating based on something the player did.

Reply:
You should make sure your other relics have a `flash()` call at appropriate times too. It looks like a few do, but mostly where you took code from the base game (e.g. Broken Matryoshka on chest open).